CHANGELOG.next.asciidoc

```diff
@@ -10,75 +10,27 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
-- Fix parsing of Elasticsearch node name by `elasticsearch/slowlog` fileset. {pull}14547[14547]
-- With the default configuration the cloud modules (aws, azure, googlecloud, o365, okta)
-- With the default configuration the cef and panw modules will no longer send the `host`
-- Add `while_pattern` type to multiline reader. {pull}19662[19662]
-- auditd dataset: Use process.args to store program arguments instead of auditd.log.aNNN fields. {pull}29601[29601]
-- Remove deprecated old awscloudwatch input name. {pull}29844[29844]
 
 *Heartbeat*
 
-- Add fonts to support more different types of characters for multiple languages. {pull}29606[29861]
 
 *Metricbeat*
 
-- Remove deprecated fields in Docker module. {issue}11835[11835] {pull}27933[27933]
-- Remove deprecated fields in Kafka module. {pull}27938[27938]
-- Remove deprecated config option default_region from aws module. {pull}28120[28120]
-- Remove network and diskio metrics from ec2 metricset. {pull}28316[28316]
-- Rename read/write_io.ops_per_sec to read/write.iops in rds metricset. {pull}28350[28350]
-- Remove linux-only metrics from diskio, memory {pull}28292[28292]
-- Remove deprecated config option perfmon.counters from windows/perfmon metricset. {pull}28282[28282]
-- Remove deprecated fields in Redis module. {issue}11835[11835] {pull}28246[28246]
-- system/process metricset: Replace usage of deprecated `process.ppid` field with `process.parent.pid`. {pull}28620[28620]
 
- *Packetbeat*
+*Packetbeat*
 
-- Redis: fix incorrectly handle with two-words redis command. {issue}14872[14872] {pull}14873[14873]
-- `event.category` no longer contains the value `network_traffic` because this is not a valid ECS event category value. {pull}20556[20556]
-- Remove deprecated TLS fields in favor of tls.server.x509 and tls.client.x509 ECS fields. {pull}28487[28487]
-- HTTP: The field `http.request.method` will maintain its original case. {pull}28620[28620]
-- Unify gopacket dependencies. {pull}29167[29167]
 
 *Winlogbeat*
 
-- Add support to Sysmon file delete events (event ID 23). {issue}18094[18094]
-- Improve ECS field mappings in Sysmon module. `related.hash`, `related.ip`, and `related.user` are now populated. {issue}18364[18364]
-- Improve ECS field mappings in Sysmon module. Hashes are now also populated to the corresponding `process.hash`, `process.pe.imphash`, `file.hash`, or `file.pe.imphash`. {issue}18364[18364]
-- Improve ECS field mappings in Sysmon module. `file.name`, `file.directory`, and `file.extension` are now populated. {issue}18364[18364]
-- Improve ECS field mappings in Sysmon module. `rule.name` is populated for all events when present. {issue}18364[18364]
-- Remove top level `hash` property from sysmon events {pull}20653[20653]
-- Move module processing from local Javascript processor to ingest node {issue}29184[29184] {pull}29435[29435]
 - Fix run loop when reading from evtx file {pull}30006[30006]
```

Comment:
@leehinman @andrewkroh this change did not make it into the final BC but it is listed as a breaking change. Looking at the PR it looks like a normal bugfix, should I move it to the appropriate section?

Reply:
Yes, that's a bugfix.